config/global.ini.php

```diff
@@ -392,11 +392,15 @@
 ; This feature will not work with the MYSQLI extension.
 archiving_query_max_execution_time = 7200
 
-
 ; Allows you to disable archiving segments for selected plugins. For more details please see https://matomo.org/faq/how-to-disable-archiving-the-segment-reports-for-specific-plugins
 ; Here you can specify the comma separated list eg: "plugin1,plugin2"
 disable_archiving_segment_for_plugins = ""
 
+; By default Matomo will archive data showing the contribution of each action to goal conversions, for sites tracking millions
+; of visits with a large number of goals this may negatively impact archiving performance. You can disable archiving of action
+; goal contribution here:
+disable_archive_actions_goals = 0
```

Comment:
Looks like you've got a bit of code from your other PR in here?

Reply:
It was originally branched off `4.x-dev`, I did rebase to `next_release` though, so not quite sure why those other commits are being included.